vehicles-utils/upload-documents/upload-documents.py

```
@@ -0,0 +1,113 @@
+    """Obtiene una lista de archivos en un directorio y sus subdirectorios."""
+    files = []
+    for root, _, filenames in os.walk(directory):
+        for filename in filenames:
+            full_path = os.path.join(root, filename)
+            if os.path.isfile(full_path):
+                files.append(
+                    (
+                        "files",
+                        (
+                            filename,
+                            open(full_path, "rb"),
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)